### DISCUSSION TITLE
Question: Does WinUI still support RS2 or should/will this change soon?

### DISCUSSION THREAD

**Discussion by marcelwgn** [microsoft/microsoft-ui-xaml#7517] [Q&A]
Given that RS2 is no longer a supported OS build (went out of support October last year), does WinUI 2 still support RS2? Should WinUI still support this version?

@kmahone mentioned [here](https://github.com/microsoft/microsoft-ui-xaml/pull/2809) that it is not supported anymore, so should we update the readme and code to reflect that?

**Comment by StephenLPeters** [comment]
@kmahone Is it true that we aren't supporting RS2 anymore? @ryandemopoulos and maybe @jevansaks what criteria do we use to make this decision?

**Comment by maxkatz6** [comment]
I want to note, that there are long-term support windows versions:
Version | Servicing option | Mainstream support end date
-- | -- | --
1809 | Long-Term Servicing Channel (LTSC) | 2024-01-09
1607 | Long-Term Servicing Branch (LTSB) | 2021-10-12
1507 (RTM) | Long-Term Servicing Branch (LTSB) | 2020-10-13

And some developers need to support these versions in their applications.
However WinUI already doesn't support 1607. So seems it doesn't claims to support actually supported versions at all.
But it still better than [WCT situation](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Directory.Build.props#L18) that never was available for some applications because of principle "All features will be supported for two Windows SDK for Windows 10 release cycles or until another principle supersedes it"

**Comment by maxkatz6** [comment]
From other hand, on the Build I heard that WinUI 3 will probably support wider list of Windows versions including Win8.
I haven't read detailed information about it tho.

**Comment by Felix-Dev** [comment]
@maxkatz6 
> From other hand, on the Build I heard that WinUI 3 will probably support wider list of Windows versions including Win8.

Might be better to scratch that "probably" for now regarding Windows 8. The team has made no promises whatsoever regarding Windows 8. And while more downlevel support for older Windows 10 versions might arrive further down the line, for now, WinUI 3 is only publicly stated to support Windows 10 1803 and above.

**Comment by kmgallahan** [comment]
@maxkatz6 

RS2 = build 15063 / version 1703, which isn't LTSB/LTSC

**Comment by jevansaks** [comment]
We're still supporting RS2 for now since the incremental cost on WinUI to do so is low. As we go forward into WinUI3 we'll need to re-evaluate that. But for now I expect that WinUI 2.x can continue to support RS2.

**Comment by michael-hawker** [comment]
Is there an update on this now with more WinUI 3 plans solidified?

The Readme is currently out-of-date mentioning 2.4 supporting RS2, but nothing about the 2.5 release.

Is the plan to make a bump for 2.6 or later in the future to help get folks ready to move to WinUI 3?

I know in the Toolkit we just bumped to RS5 since it's the next LTSC release and the rest of standard releases will be out of support as of May.
